File: Bubble_Sort/bubble_sort.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int n = 6;
    int arr[] = {6, 1, 9, 2, 1, 9};
    for(int i = 0; i < n; i++){
        for(int j = 0; j < n-i-1; j++){
            if(arr[j] <= arr[j+1]){
                swap(arr[j], arr[j+1]);
            }
        }
    }

    for(int i= 0; i < n; i++){
        cout << arr[i] << " ";
    }
    return 0;
}

File: Bubble_Sort/bubble_sort.py
arr = [6, 1, 9, 2, 1, 9]
n = 6

for i in range(n):
    for j in range(n-i-1):
        if arr[j] <= arr[j+1]:
            arr[j], arr[j+1] = arr[j+1], arr[j]

print(arr)